From de858b1e855ecb818d53416834cc41db20211026 Mon Sep 17 00:00:00 2001
From: Gautham Kumar Senthilkumaran <quic_gauthamk@quicinc.com>
Date: Wed, 20 Sep 2023 22:14:41 +0530
Subject: [PATCH] mac80211: Change in nl80211 enum for 6GHz operation

Currently there is change in latest code and aplied patch.
So added this temporary patch file for 6GHz working.

Signed-off-by: Gautham Kumar Senthilkumaran <quic_gauthamk@quicinc.com>
---
 include/uapi/linux/nl80211.h | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

--- a/include/uapi/linux/nl80211.h
+++ b/include/uapi/linux/nl80211.h
@@ -3404,27 +3404,20 @@ enum nl80211_attrs {
 	NL80211_ATTR_RX_HW_TIMESTAMP,
 	NL80211_ATTR_TD_BITMAP,
 
-	NL80211_ATTR_PUNCT_BITMAP,
-
-	NL80211_ATTR_MAX_HW_TIMESTAMP_PEERS,
-	NL80211_ATTR_HW_TIMESTAMP_ENABLED,
-
-	NL80211_ATTR_EMA_RNR_ELEMS,
-
-	NL80211_ATTR_MLO_LINK_DISABLED,
-
-	NL80211_ATTR_WIPHY_ANTENNA_GAIN,
-
 	NL80211_ATTR_HE_MUEDCA_PARAMS,
 
 	NL80211_ATTR_BEACON_TX_MODE,
 
+	NL80211_ATTR_WIPHY_ANTENNA_GAIN,
+
 	NL80211_ATTR_STA_MGMT_RTS_CTS_CONFIG,
 
 	NL80211_ATTR_AWGN_INTERFERENCE_BITMAP,
 
 	NL80211_ATTR_6G_REG_POWER_MODE,
 
+	NL80211_ATTR_EMA_RNR_ELEMS,
+
 	NL80211_ATTR_RU_PUNCT_SUPP_BW,
 
 	NL80211_ATTR_RU_PUNCT_SUPP_HE,
@@ -3443,6 +3436,13 @@ enum nl80211_attrs {
 	NL80211_ATTR_DEL_MULTI_CHAN,
 	/* add attributes here, update the policy in nl80211.c */
 
+	NL80211_ATTR_PUNCT_BITMAP,
+
+	NL80211_ATTR_MAX_HW_TIMESTAMP_PEERS,
+	NL80211_ATTR_HW_TIMESTAMP_ENABLED,
+
+	NL80211_ATTR_MLO_LINK_DISABLED,
+
 	__NL80211_ATTR_AFTER_LAST,
 	NUM_NL80211_ATTR = __NL80211_ATTR_AFTER_LAST,
 	NL80211_ATTR_MAX = __NL80211_ATTR_AFTER_LAST - 1
